plugins/micromega/g_micromega.mlg

```diff
@@ -32,6 +41,14 @@ END
 
 
 
+
+
+TACTIC EXTEND ITER
+| [ "iter" tactic(t) ] -> { iter_hyps t }
```

Comment:
This seems much more general and useful. It should be moved to a better place.

Reply:
I don't think this should be exported as a built-in tactic, this is going to add parsing rules for no good reasons. I know that Ltac2 didn't make it into 8.10 (for bad reasons), but that's not a justification good enough to introduce crappy half-baked combinators when what is missing is a first-class notion of lists in Ltac1.